src/SFA.DAS.ApprenticeFeedback.Application/Commands/ProcessEmailTransaction/ProcessEmailTransactionCommandHandler.cs

```diff
@@ -57,7 +58,7 @@ public async Task<ProcessEmailTransactionResponse> Handle(ProcessEmailTransactio
                     different actions need to be taken to populate the das-notifications SendEmailCommand 
                     from the notifications package.)
 
-                Update the transaction with relevant information about who it was sent to,
+                Update the trans action with relevant information about who it was sent to,
```

Comment:
Was the splitting of the word transaction in the comment intentional?

Reply:
It was not, i've reverted it